opencost/src/index.tsx

```diff
@@ -16,9 +16,17 @@ import { Settings } from './settings';
 import { getConfigStore, getDisplayCurrency, getDisplayTimespan, getServiceDetails } from './utils';
 
 function IconAction() {
+  const handleClick = event => {
+    event.preventDefault();
+    const targetElement = document.getElementById('cost');
+    if (targetElement) {
+      targetElement.scrollIntoView({ behavior: 'auto' });
+    }
+  };
+
   return (
     <Tooltip title={'Cost'}>
-      <IconButton href="#cost" aria-label={'Cost'} aria-haspopup="true">
+      <IconButton onClick={handleClick} href="#cost" aria-label={'Cost'} aria-haspopup="true">
```

Comment:
If the href still needed?

Reply:
While we're still at it, can you rename the anchor ID to something more specific, to avoid clashes with other eventual plugin generated UI? something like `opencost-plugin-cost-section`?

Reply:
Fixed it